src/api/index.js

```diff
@@ -51,17 +51,21 @@ const useApi = () => {
     return resp?.data?.data?.name || ''
   }
 
-  const getImageSrc = (uuid, dim) =>
+  const getImageSrc = (uuid, dim, scale = 'fit') =>
     uuid && uuid !== '[]'
-      ? `${IMAGE_URL}/${dim ? `${dim.width}x${dim.height || ''},fit/` : ''}${MEDIA_URL}/${uuid}`
+      ? `${IMAGE_URL}/${
+          dim ? `${dim.width}x${dim.height || ''},${scale}/` : ''
+        }${MEDIA_URL}/${uuid}`
       : null
 
   const getDocumentSrc = uuid => (uuid && uuid !== '[]' ? `${MEDIA_URL}/${uuid}` : null)
 
-  const getImageSrcList = (uuidList, dim) =>
+  const getImageSrcList = (uuidList, dim, scale = 'fit') =>
```

Comment:
Don't default it to scale='fit'. We can have it as a parameter and pass it as an argument any time we need it but let's not default it to fit.

Reply:
I did that as there are other uses of this method, and as originally it was hard-coded to fit, I believed it was the simplest approach to prevent issues

Reply:
I have now updated it to not have a default, see latest changes